aero.minova.core.application.system.service/src/main/java/aero/minova/core/application/system/controller/SqlProcedureController.java

```diff
@@ -124,9 +127,14 @@ public ResponseEntity executeProcedure(@RequestBody Table inputTable) throws Exc
 			}
 			if (extensions.containsKey(inputTable.getName())) {
 				try {
+					/*
+					 * Extensions können auch Reqeusts an Dienste beinhalten, welche sich in die Queere kommen können. Deshalb sollten diese durch eine
```

Comment:
Bitte jeden Satz in eine eigen Zeile schreiben.
Vor allem, wenn der vorherige etwas länger ist.